examples/community/lpw_stable_diffusion.py

```diff
@@ -488,24 +488,6 @@ def __init__(
             feature_extractor=feature_extractor,
         )
 
-    def enable_xformers_memory_efficient_attention(self):
```

Comment:
here and below: inherits from `DiffusionPipeline` so I figured that this could be defined there (with the recursive take) to remove a lot of code duplication